.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
I think features and feature requests should be the same thing. And this template is good enough. If someone doesn't know the implementation details, then that field can be omitted (and it frequently is).

Reply:
Done. I removed feature request category and update comment regarding implementation details.